lib/Dialect/TritonGPU/Transforms/Combine.cpp

```diff
@@ -560,19 +561,46 @@ class RematerializeForward : public mlir::RewritePattern {
 // -----------------------------------------------------------------------------
 //
 // -----------------------------------------------------------------------------
+static int computeCapabilityToMMAVersion(int computeCapability) {
+  if (computeCapability < 80) {
+    return 1;
+  } else if (computeCapability < 90) {
+    return 2;
+  } else {
+    assert(computeCapability < 90 && "computeCapability > 90 not supported");
+    return 0;
+  }
+}
+
+static SmallVector<int64_t, 2>
+mmaVersionToShapePerWarp(int version, const ArrayRef<int64_t> &shape,
```

Comment:
I see `ArrayRef &` has been used in other places. Probably you can clean it up later.

Reply:
not get your mean.

Reply:
Thanks, I got.

Reply:
this is because here: [getWarpsPerTile](https://github.com/openai/triton/blob/triton-mlir/lib/Dialect/TritonGPU/Transforms/Combine.cpp#L570)

I don't want to break this.

Reply:
Yeah, I noticed that. Maybe clean up the code in another PR